src/main/java/com/osiris/autoplug/client/tasks/updater/mods/TaskModsUpdater.java

```diff
@@ -231,8 +233,8 @@ public void runAtStart() throws Exception {
                 String type = result.getDownloadType(); // The file type to download (Note: When 'external' is returned nothing will be downloaded. Working on a fix for this!)
                 String latest = result.getLatestVersion(); // The latest version as String
                 String downloadUrl = result.getDownloadUrl(); // The download url for the latest version
-                String resultmodrinthId = result.getSpigotId();
```

Comment:
Shouldnt this be from result instead of mod?

Reply:
`mod` is set to `result.mod` here. If I'm not mistaken, it's the only place the Modrinth ID is stored in the `result`.